docs/api/main-config-babel.md

```diff
@@ -0,0 +1,102 @@
+---
+title: 'config.babel'
+---
+
+<!-- TODO: Is `babel` or `babelDefault` the one we prefer? -->
```

Comment:
I found both `babel` and `babelDefault` (with identical types) in the types. Which is preferred? Do we need to document both?

Reply:
Yeah addon creators should use `babelDefault` and `main.ts` can simply use `babel`, to ensure the user has a final say.